password-mode.el

```diff
@@ -44,7 +44,8 @@
 
 Regexps must not contain parentheses for grouping, otherwise your
 match wouldn't work.  Shy groups are OK."
-  :type '(repeat (regexp :tag "Password Regex")))
+  :type '(repeat (regexp :tag "Password Regex"))
+  :group 'password-mode)
```

Comment:
Forgive the intrusion, just curious: is this required?  The elisp manual says, `If a defcustom does not specify any :group, the last group defined with defgroup in the same file will be used. This way, most defcustom do not need an explicit :group.`